src/renderer/qss/js/qssServiceButtons.js

```diff
@@ -24,10 +24,10 @@
      * QSS button.
      */
     fluid.defaults("gpii.qss.keyInButtonPresenter", {
-        gradeNames: ["gpii.qss.buttonPresenter"],
+        gradeNames: ["gpii.qss.disabledButtonPresenter"],
         attrs: {
             "aria-label": "Settings Panel"
-        },
+        }/*,
         listeners: {
```

Comment:
Please add comment here indicating that this was commented out for GPII-3773 and may shortly be removed completely